tensorflow/core/util/port.cc

```diff
@@ -64,12 +64,14 @@ bool GpuSupportsHalfMatMulAndConv() {
 
 // Returns whether oneDNN should be enabled or disabled by default.
 //   - Linux: Enabled by default for CPUs with neural network features.
-//   - Windows: Disabled by default.
+//   - Windows: Enabled by default.
```

Comment:
Please also update the comment.
```suggestion
//   - Windows: Enabled by default on x86 CPUs.
```